crates/engine/src/lib.rs

```diff
@@ -7,6 +7,8 @@ pub mod io;
 
 use anyhow::{bail, Context, Result};
 use io::IoStreamRedirects;
+use spin_config::host_component::ComponentConfig;
+use spin_config::Resolver;
```

Comment:
nit: merge the two statements.

Reply:
Is there a tool we can use to enforce one `use` style?

Reply:
I'd be fine with https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#crate

Reply:
Oh its unstable-only :angry: